src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/AlertSetting/GetAzureRmRecoveryServicesAsrAlertSetting.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ----------------------------------------------------------------------------------
+// 
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Management.RecoveryServices.SiteRecovery.Models;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery
+{
+    /// <summary>
+    /// Gets Azure Site Recovery Notification / Alert Settings.
+    /// </summary>
+    [Cmdlet(
+        VerbsCommon.Get, 
+        "AzureRmRecoveryServicesAsrAlertSetting")]
+    [Alias(
+        "Get-ASRNotificationSetting",
+        "Get-AzureRmRecoveryServicesAsrNotificationSetting",
+        "Get-ASRAlertSetting")]
+    [OutputType(typeof(IEnumerable<ASRAlertSetting>))]
+    public class GetAzureRmRecoveryServicesAsrAlertSetting : SiteRecoveryCmdletBase
```

Comment:
@cormacpayne   These are not actual alert , these are the settings which defines the alert . And Setting is at SUBSCIPTION level.

Alert / Notifications are send as Mail .

Commet From ASR Service 
                 // This is done in this manner because this entity is a singleton in the vault.
                // There can only be one alert setting in the system. It has to be with the
                // default name. In case the user tries to fetch an alert setting with a
                // different name, he will get a 404.
  
We  can discuss we can do something about it.